src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -351,9 +351,10 @@ public void mergeAlignment(final File referenceFasta) {
 
             // See if there are alignments for current unaligned read or read pair.
             if (nextAligned != null && rec.getReadName().equals(nextAligned.getReadName())) {
-                // If there are multiple alignments for a read (pair), then the unaligned SAMRecord must be cloned
-                // before copying info from the aligned record to the unaligned.
-                final boolean clone = nextAligned.numHits() > 1 || nextAligned.hasSupplementalHits();
+                // If there are multiple alignments for a read (pair), then the unaligned SAMRecord must be deep copied
+                // before copying info from the aligned record to the unaligned. Cloning in htsjdk is not deep enough
+                // since the bases can be reverse complemented in the case where one read aligns to both strands.
```

Comment:
Perhaps say "...since later (below) the bases can be...".  I was confused if it meant prior to or after this statement.